app/grants/templates/grants/new.html

```diff
@@ -56,7 +56,7 @@
         <div id="robot_error" class="hidden">
           <img src="{% static "v2/images/prime.png" %}">
         </div>
-        <div class="col-sm-12 col-md-10 col-lg-6" id="primary_form">
+        <div class="col-sm-12 col-md-10 col-lg-6" id="grants_form">
```

Comment:
Should we update the other references to `primary_form` throughout `grants/*` ?

Reply:
@SaptakS `assets/v2/css/grants/new.css`

Reply:
Done. :)